sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -503,7 +503,7 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
    */
   override def databaseExists(dbName: String): Boolean = {
     try {
-      getDatabase(dbName)
+      makeDatabase(None, dbName)
```

Comment:
I disagree with this change. There is nearly no benefit and it's kind of encouraging people to never use parameter default values but just call the base function with all parameter values specified.

Reply:
SGTM. Let's revert it.